### ISSUE TITLE
Support for Onlyfans

### ISSUE THREAD

**Issue by IJCS** [ytdl-org/youtube-dl#24465]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2020.03.24**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

-Free post https://onlyfans.com/16391013/killercleavage-free
-Free post https://onlyfans.com/15584274/hoodedhofree
-Free post https://onlyfans.com/13634727/lilianlacefree
-Paid post https://onlyfans.com/16352300/entina_cat
-Free post https://onlyfans.com/16384958/silvermoonfree

## Description

I think it would be very helpful to add this to the page list. Especially for mass backups of accounts disconnected from the internet. Always protecting content using credentials.

**Comment by marium0505** [comment]
You do realize that a lot of content on OF is copyrighted and that they (the creators) ask people not to download their content?

**Comment by interbiznw** [comment]
> You do realize that a lot of content on OF is copyrighted and that they (the creators) ask people not to download their content?

Yes as are other sites youtube-dl can download from. That doesn't automatically assume that it will be used for purposes against the terms of use. As the OP said, it can be used for mass backups/archival purposes of already uploaded videos by the user. People lose their originals all the time and would be a good way to retrieve a copy at the highest format available.

Not to mention, this clearly can already be done by scraping the urls individually in chrome dev tools like any other site.

**Comment by laichiaheng** [comment]
Any news?
